### PR TITLE
Fetch linkedin contacts

### PR DESCRIPTION
**Pull request by valerykalashnikov**
Fetch contacts from Linkedin network. Fields: id, first_name, last_name, name, picture-url

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,13 +33,14 @@ require "omnicontacts"
 Rails.application.middleware.use OmniContacts::Builder do
   importer :gmail, "client_id", "client_secret", {:redirect_path => "/oauth2callback", :ssl_ca_file => "/etc/ssl/certs/curl-ca-bundle.crt"}
   importer :yahoo, "consumer_id", "consumer_secret", {:callback_path => '/callback'}
+  importer :yahoo, "consumer_id", "consumer_secret", {:callback_path => "/contacts/linkedin/callback", :state => '<long_unique_string_value>'}
```
Comment (rubytastic) [inline README.md]: :yahoo should become :linkedin ? seems wrong

Comment (valerykalashnikov) [inline README.md]: I was a little bit mixed up. Of course, :redirect_path option works.

**Comment by valerykalashnikov** [comment]
I fixed how to use linked in in Readme. Of course there should be :redirect_path option instead of :callback_path because linkedin authorises via Oauth2.

**Comment by rubytastic** [comment]
Basically we would like to have solid tests before we merge these bigger changes.

Could you please provide an update to the tests before we merge this in?

**Comment by valerykalashnikov** [comment]
Sure, but I should know which kind of tests do you mean? 
As I saw in coverage/index.html the weak place is http_utils tests. Do you mean this one or something different?

**Comment by xacaxulu** [comment]
@valerykalashnikov you should provide specs that cover (test against) any code that you added, written in RSpec. After running `rspec spec` you can view the coverage report to make sure that your code has been fully tested, line by line, and that all the specs pass. This will prove that your code works as expected and will give the maintainers confidence to merge your code into master.

**Comment by valerykalashnikov** [comment]
Thanks, @xacaxulu! I've pushed spec for linkedin importer https://github.com/valerykalashnikov/omnicontacts/blob/linkedin-support/spec/omnicontacts/importer/linkedin_spec.rb based on previous importer specs and I don't know what's wrong with it. Coverage report speak 94.59 % 
Could you drop line-by-line comment to say me what's wrong with my spec?
(It's my first pullrequest to the opensource)

**Comment by rubytastic** [comment]
Any updates on this? I can merge if test are OK.
Could be merged in but not fully tested is not the way you normally want to go.
I merge for now and let ticket be open,
please provide any issues with linkedin import afterwards.

**Pull request merged**